.github/workflows/jobs.yml

```diff
@@ -185,7 +185,7 @@ jobs:
         run: ./mirror-registry uninstall -u jonathan -H quay --autoApprove -v -k /home/runner/.ssh/id_rsa
 
       - name: Download old mirror-registry tarball that runs quay with postgres
-        run: wget -O mirror-registry-postgres.tar.gz https://github.com/quay/mirror-registry/releases/download/v1.3.10/mirror-registry-online.tar.gz
+        run: wget -O mirror-registry-postgres.tar.gz https://github.com/quay/mirror-registry/releases/download/v1.3.10/mirror-registry-offline.tar.gz
```

Comment:
this has to be parsed with `${{ matrix.installer-type }}`

Reply:
good catch!